ptf_tests/tests_to_run_BAT.txt

```diff
@@ -0,0 +1,12 @@
+l3_action_selector_with_tap_ports:config_json='l3_action_selector_with_tap_test1.json'
```

Comment:
Can we change this to md or rst file, add a license and have a short description of what the file contains